lib/releaselib.sh

```diff
@@ -397,31 +397,35 @@ release::set_release_version () {
 }
 
 ###############################################################################
-# Create GCS bucket for releasing Kube if necessary. Ensure that the default
+# Create GCS bucket for publishing. Ensure that the default
 # ACL allows public reading of artifacts.
+#
+# Insufficient for checking actual writability, but useful:
+# ganpati list-members -g cloud-kubernetes-release -u $USER|fgrep -wq $USER
+#
 # @param bucket - The gs release bucket name
 # @return 1 if bucket can't be made
 release::gcs::ensure_release_bucket() {
   local bucket=$1
-  local current_defacl
-  local new_acl_file
+  local tempfile=/tmp/$PROG-gcs-write.$$
 
   if ! $GSUTIL ls "gs://$bucket" >/dev/null 2>&1 ; then
     logecho -n "Creating Google Cloud Storage bucket $bucket: "
     logrun -s $GSUTIL mb -p "$GCLOUD_PROJECT" "gs://$bucket" || return 1
-    logecho -n "Adding public-read default ACL on bucket $bucket: "
-    logrun -s $GSUTIL defacl ch -u AllUsers:R "gs://$bucket" || return 1
   fi
 
-  if [[ $($GSUTIL defacl get "gs://$bucket" 2>/dev/null | python -c '\
-          import sys,json;\
-          print " ".join(x["role"] for x in json.load(sys.stdin) if x["entity"] == "allUsers")')\
-        != 'READER' ]]; then
-    logecho "GCS bucket $bucket is missing default public-read ACL,"
-    logecho "or you lack permission to see the default ACL."
-    logecho "Please run"
-    logecho "  gsutil defacl ch -u AllUsers:R gs://$bucket"
-    logecho "and try again."
+  logecho -n "Ensure public-read default ACL on bucket $bucket: "
+  logrun -s $GSUTIL defacl ch -u AllUsers:R "gs://$bucket" || return 1
```

Comment:
I think I'd previously made this a check, rather than a fix, so that users running this explicitly had to decide that it was OK to make the bucket public-by-default.

It might be OK to change this behavior, though.